tests/src/baseservices/exceptions/regressions/Dev11/147911/fpcw.cpp

```diff
@@ -23,6 +23,8 @@ LONG WINAPI ExcepFilter(struct _EXCEPTION_POINTERS *pExp)
 
 
 // This is an example of an exported function.
+// Disable optimization otherwise compiler removes dividebyzero operation
```

Comment:
testcode at https://github.com/dotnet/coreclr/blob/master/tests/src/baseservices/exceptions/regressions/Dev11/147911/fpcw.cpp#L46 is trying to cause a floating point divide by zero exception which is handled by the test. However the statements used to cause divide-by-zero exception can be removed by an optimizing compiler as being useless. This change forces the compiler to not do that.